test/test_browser.py

```diff
@@ -3320,10 +3324,17 @@ def test_cocos2d_hello(self):
                      '-Wno-inconsistent-missing-override',
                      '-Wno-deprecated-declarations'])
 
-  def test_async(self):
+  @parameterized({
+    'asyncify': (['-sASYNCIFY=1'],),
+    'jspi': (['-sASYNCIFY=2', '-Wno-experimental'],),
+  })
```

Comment:
If we add more of these it might make sense to add a `@also_with_jspi` decorator or such, like we have for other stuff, but I'm not sure. Just a thought for later.

Reply:
Yeah, I was going to add this, but wasn't sure what happens when you have multiple `-sASYNCIFY` args i.e. the main test will have `-sASYNCIFY=1`, but the `@also_with_jspi` will add `-sASYNCIFY=2`. I'm guessing the last overwrites it?

Reply:
Yes, later `-s` flags should override.